app/components/custom-source-properties/PlatformAppProperties.vue.ts

```diff
@@ -15,16 +16,18 @@ export default class PlatformAppProperties extends Vue {
   @Inject() navigationService: NavigationService;
   @Inject() platformAppsService: PlatformAppsService;
   @Inject() windowsService: WindowsService;
+  @Inject() sideNavService: SideNavService;
 
   get managerSettings() {
     return this.source.getPropertiesManagerSettings() as IPlatformAppManagerSettings;
   }
 
-  navigateApp() {
+  navigateApp(appId: string) {
     this.navigationService.navigate('PlatformAppMainPage', {
       appId: this.appId,
       sourceId: this.source.sourceId,
     });
+    this.sideNavService.setCurrentMenuItem(appId);
```

Comment:
we already have access to `this.appId` here, not sure why we need it again as an argument 🤔

Reply:
Palm face moment for sure